src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -68,4 +71,10 @@ public static Set<Tag> getTagSet(String... strings) {
                 .collect(Collectors.toSet());
     }
 
+    public static ReadOnlyCourseName getSampleCourseName() {
+        CourseName courseName = new CourseName();
+        courseName.setCourse(new Course("XX1234Y"));
```

Comment:
Perhaps we can use a constant I suggested in the earlier comment here, instead of a magic literal:
```suggestion
        courseName.setCourse(new Course(COURSE_CODE_FORMAT));
```